crates/blockifier/src/execution/syscalls/syscall_tests/library_call.rs

```diff
@@ -134,12 +134,12 @@ fn test_nested_library_call(test_contract: FeatureContract, expected_gas: u64) {
         class_hash: Some(test_class_hash),
         code_address: None,
         call_type: CallType::Delegate,
-        initial_gas: if_sierra(9999874550, 9999823550),
+        initial_gas: if_sierra(9999813550, 9999823550),
         ..trivial_external_entry_point_new(test_contract)
     };
     let storage_entry_point = CallEntryPoint {
         calldata: calldata![felt!(key), felt!(value)],
-        initial_gas: if_sierra(9999874550, 9999656870),
+        initial_gas: if_sierra(9999752538, 9999656870),
```

Comment:
not specific to this PR, but could you please put `if_native`

Reply:
should be resolved